### ISSUE TITLE
Generalizing variant normalization beyond VCF/BGEN representations

### ISSUE THREAD

**Issue by emaxwell** [projectglow/glow#107]
The variant normalization function is currently implemented on the VCF and BGEN derived schema, which uses contigName, start, end, ref, and alt columns. However, there are often use cases for normalization of site lists (for example) that are not directly derived from a VCF or BGEN. As an example, it would be reasonable to normalize sites from an external resource such as dbSNP or ExAC. Those often have a CPRA representation (chromosome position ref alt). It would be good for normalization to work with these other representation types.

**Comment by henrydavidge** [comment]
Would providing a simple conversion between CPRA string ids and our current id representation be sufficient? Or is there something else that is necessary.

**Comment by emaxwell** [comment]
Yes, conversion to and from CPRA strings or chromosome, position, ref, alt as separate columns should be sufficient. The only other immediate caveats I can think of are:
1) this also relates to https://github.com/projectglow/glow/issues/108 since normalize is intended for VCF/BGEN schema data frames and therefore drops supplemental columns. For external data set site lists, they are rarely coming in from VCF/BGEN so should not be expected to fit this schema.
2) external data is more likely to have different chromosome representations as compared to the reference genome file. For example, contigNames "1" and "chr1" mean the same thing, or "X" and "23". We might want to include some basic consistency checking/reporting, even if it just entails listing how many sites could not be mapped to the reference.
